client/src/templates/Challenges/components/tool-panel.tsx

```diff
@@ -104,23 +104,36 @@ function ToolPanel({
         </Button>
       )}
       <Dropdown dropup>
-        <Dropdown.Toggle id={'get-help-dropdown'}>
+        <Dropdown.Toggle
+          id={'get-help-dropdown'}
+          data-playwright-test-label={'get-help-dropdown'}
```

Comment:
Just making sure we are not adding anything unnecessary
```suggestion
          data-playwright-test-label='get-help-dropdown'
```

Reply:
Done